src/main/java/org/reaktivity/specification/socks/internal/SocksFunctions.java

```diff
@@ -126,6 +227,60 @@ public String getPrefixName()
         }
     }
 
+    private static void fillInBytes(
+        byte[] addressBytes,
+        int index,
+        String ipv6Group)
+    {
+        short res = parseShort(ipv6Group, 16);
+        addressBytes[index++] = (byte) ((res >> 8) & 0xff);
+        addressBytes[index] = (byte) (res & 0xff);
+    }
+
+    private static void fillInIpv6HexCompressed(
+        String address,
+        byte[] addressBytes)
+    {
+        final Matcher ipv6Matcher = IPV6_HEX_COMPRESSED_MATCHER.get().reset(address);
+        ipv6Matcher.matches();
```

Comment:
These 2 lines can likely be removed, see comment at call site for this method.

Reply:
not removed all, because still need `ipv6Matcher` to get all groups